### PR TITLE
Port System.Text.Json triple slash to docs

### PR DESCRIPTION
**Pull request by carlossanlop**
Found with DocsPortingTool.

Also included a System.Text.RegularExpressions port (by mistake, but let's get it merged). @tannergooding  @pgovind 

@gewarren I'm targeting the net6-rc1 branch.

### CODE REVIEW

**Comment by gewarren** [comment]
Docs Build status updates of commit _[6f101fe](https://github.com/carlossanlop/dotnet-api-docs/commits/6f101fe9c68a2846537505fafa295c769b7d84ab)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[xml/System.Text.Json.Serialization/JsonConverter`1.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.json/xml/System.Text.Json.Serialization/JsonConverter%601.xml) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.text.json.serialization.jsonconverter-1?view=net-5.0&branch=pr-en-us-7094) | [Details](#user-content-664cb9827941ba96a51471babdf56a0c-0a6b9f54)
[xml/System.Text.Json/JsonSerializer.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.json/xml/System.Text.Json/JsonSerializer.xml) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializer?view=net-5.0&branch=pr-en-us-7094) | [Details](#user-content-25b53321868a50aa68fabde1de2bd6c3-0a6b9f54)
[xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.json/xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.text.json.serialization.metadata.jsonmetadataservices?view=net-6.0&branch=pr-en-us-7094) |
[xml/System.Text.Json.Serialization.Metadata/JsonObjectInfoValues`1.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.json/xml/System.Text.Json.Serialization.Metadata/JsonObjectInfoValues%601.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.text.json.serialization.metadata.jsonobjectinfovalues-1?view=net-6.0&branch=pr-en-us-7094) |
[xml/System.Text.Json.Serialization.Metadata/JsonParameterInfoValues.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.json/xml/System.Text.Json.Serialization.Metadata/JsonParameterInfoValues.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.text.json.serialization.metadata.jsonparameterinfovalues?view=net-6.0&branch=pr-en-us-7094) |
[xml/System.Text.RegularExpressions/Capture.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.json/xml/System.Text.RegularExpressions/Capture.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.text.regularexpressions.capture?view=dotnet-uwp-10.0&branch=pr-en-us-7094) |

<a id="664cb9827941ba96a51471babdf56a0c-0a6b9f54"></a>
### [xml/System.Text.Json.Serialization/JsonConverter`1.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.json/xml/System.Text.Json.Serialization/JsonConverter%601.xml)
 - **Line 0, Column 0**: **[Suggestion-[disallowed-html-tag](https://review.docs.microsoft.com/help/contribute/validation-ref/disallowed-html-tag?branch=main)]** `````HTML tag 'seealso' isn't allowed. Disallowed HTML poses a security risk and must be replaced with approved Docs Markdown syntax.`````

<a id="25b53321868a50aa68fabde1de2bd6c3-0a6b9f54"></a>
### [xml/System.Text.Json/JsonSerializer.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.json/xml/System.Text.Json/JsonSerializer.xml)
 - **Line 0, Column 0**: **[Suggestion-[disallowed-html-tag](https://review.docs.microsoft.com/help/contribute/validation-ref/disallowed-html-tag?branch=main)]** `````HTML tag 'exception' isn't allowed. Disallowed HTML poses a security risk and must be replaced with approved Docs Markdown syntax.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C3%5C500c749d-de78-3d4a-00e4-269d599bf9ec%5CPullRequest%5C202109030502234659-7094%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=H91FFTerIZd5Orz%2B1G2eRz4wCr6RskUss5IVJO8HL2o%3D&st=2021-09-03T05%3A29%3A30Z&se=2021-10-04T05%3A34%3A30Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/500c749d-de78-3d4a-00e4-269d599bf9ec?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: xml/System.Text.Json.Serialization/JsonConverter`1.xml [hunk 1]
```diff
@@ -254,11 +262,19 @@ For more information, see [How to write custom converters for JSON serialization
         <Parameter Name="options" Type="System.Text.Json.JsonSerializerOptions" Index="2" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="writer">To be added.</param>
-        <param name="value">To be added.</param>
-        <param name="options">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="writer">The <see cref="T:System.Text.Json.Utf8JsonWriter" /> to write to.</param>
+        <param name="value">The value to convert. Note that the value of <seealso cref="P:System.Text.Json.Serialization.JsonConverter`1.HandleNull" /> determines if the converter handles <see langword="null" /> values.</param>
```
Comment (carlossanlop) [inline xml/System.Text.Json.Serialization/JsonConverter`1.xml:266]:
```suggestion
        <param name="value">The value to convert. Note that the value of <see cref="P:System.Text.Json.Serialization.JsonConverter`1.HandleNull" /> determines if the converter handles <see langword="null" /> values.</param>
```

File: xml/System.Text.Json/JsonSerializer.xml [hunk 2]
```diff
@@ -2610,12 +2754,18 @@ For more information, see [How to serialize and deserialize JSON](/dotnet/standa
...
+        <param name="options">Options to control the conversion behavior.</param>
+        <summary>Converts the provided value into a <see cref="T:System.Text.Json.JsonDocument" />.</summary>
+        <returns>A <see cref="T:System.Text.Json.JsonDocument" /> representation of the value.</returns>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentException">
+          <paramref name="inputType" /> is not compatible with <paramref name="value" />.</exception>
+        <exception cref="T:System.NotSupportedException">
+          <exception cref="T:System.ArgumentNullException">
+            <paramref name="inputType" /> is <see langword="null" />.
+            </exception> There is no compatible <see cref="T:System.Text.Json.Serialization.JsonConverter" /> for <paramref name="inputType" />  or its serializable members.</exception>
```
Comment (carlossanlop) [inline xml/System.Text.Json/JsonSerializer.xml:2768]:
```suggestion
        <exception cref="T:System.ArgumentNullException">
            <paramref name="inputType" /> is <see langword="null" />.
        </exception>
        <exception cref="T:System.NotSupportedException">There is no compatible <see cref="T:System.Text.Json.Serialization.JsonConverter" /> for <paramref name="inputType" />  or its serializable members.</exception>
```

**Comment by gewarren** [comment]
Docs Build status updates of commit _[04e2243](https://github.com/carlossanlop/dotnet-api-docs/commits/04e224319bfa184026ec03cc85694da59c391d60)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.json/xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.text.json.serialization.metadata.jsonmetadataservices?view=net-6.0&branch=pr-en-us-7094) |
[xml/System.Text.Json.Serialization.Metadata/JsonObjectInfoValues`1.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.json/xml/System.Text.Json.Serialization.Metadata/JsonObjectInfoValues%601.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.text.json.serialization.metadata.jsonobjectinfovalues-1?view=net-6.0&branch=pr-en-us-7094) |
[xml/System.Text.Json.Serialization.Metadata/JsonParameterInfoValues.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.json/xml/System.Text.Json.Serialization.Metadata/JsonParameterInfoValues.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.text.json.serialization.metadata.jsonparameterinfovalues?view=net-6.0&branch=pr-en-us-7094) |
[xml/System.Text.Json.Serialization/JsonConverter`1.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.json/xml/System.Text.Json.Serialization/JsonConverter%601.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.text.json.serialization.jsonconverter-1?view=net-5.0&branch=pr-en-us-7094) |
[xml/System.Text.Json/JsonSerializer.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.json/xml/System.Text.Json/JsonSerializer.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializer?view=net-5.0&branch=pr-en-us-7094) |
[xml/System.Text.RegularExpressions/Capture.xml](https://github.com/carlossanlop/dotnet-api-docs/blob/upd.json/xml/System.Text.RegularExpressions/Capture.xml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/system.text.regularexpressions.capture?view=dotnet-uwp-10.0&branch=pr-en-us-7094) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C3%5C500c749d-de78-3d4a-00e4-269d599bf9ec%5CPullRequest%5C202109030624326274-7094%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=siwsnl39DLjCD81McipXKv0J8v7UnTC47VP575l8r0c%3D&st=2021-09-03T06%3A52%3A24Z&se=2021-10-04T06%3A57%3A24Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/500c749d-de78-3d4a-00e4-269d599bf9ec?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by eiriktsarpalis — APPROVED** [review]
Thanks. This should address the final batch of APIs that were added in RC1.

Fixes https://github.com/dotnet/runtime/issues/54862

**Comment by carlossanlop** [comment]
> Thanks. This should address the final batch of APIs that were added in RC1.

Actually, not yet. There are some more APIs in my report that do not have triple slash. Mostly from System.Text.Json.Nodes.

There's the possibility that the DocsPortingTool is unable to match the DocId of those APIs. Could you help me figure out if it's due to lack of docs or a bug in the tool? @eiriktsarpalis

File: xml/System.Text.RegularExpressions/Capture.xml [hunk 3]
```diff
@@ -335,8 +335,8 @@
         <ReturnType>System.ReadOnlySpan&lt;System.Char&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the captured span from the input string.</summary>
```
Comment (pgovind) [inline xml/System.Text.RegularExpressions/Capture.xml:338]: LGTM

File: xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml [hunk 4]
```diff
@@ -821,12 +821,13 @@
...
-        <param name="objectInfo">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="T">The type of the class or struct.</typeparam>
+        <param name="options">The <see cref="T:System.Text.Json.JsonSerializerOptions" /> to initialize the metadata with.</param>
+        <param name="objectInfo">Provides serialization metadata about an object type with constructors, properties, and fields.</param>
+        <summary>Creates metadata for a complex class or struct.</summary>
+        <returns>A <see cref="T:System.Text.Json.Serialization.Metadata.JsonTypeInfo`1" /> instance representing the class or struct.</returns>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException">Thrown when <paramref name="options" /> or <paramref name="objectInfo" /> is <see langword="null" />.</exception>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonMetadataServices.xml:830]:
```suggestion
        <exception cref="T:System.ArgumentNullException">
          <paramref name="options" /> or <paramref name="objectInfo" /> is <see langword="null" />.</exception>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonObjectInfoValues`1.xml [hunk 5]
```diff
@@ -76,7 +76,7 @@
         <ReturnType>System.Text.Json.Serialization.JsonNumberHandling</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Specifies how number properties and fields should be processed when serializing and deserializing.</summary>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonObjectInfoValues`1.xml:79]:
```suggestion
        <summary>Gets or sets an object that specifies how number properties and fields should be processed when serializing and deserializing.</summary>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonObjectInfoValues`1.xml [hunk 6]
```diff
@@ -97,7 +97,7 @@
         <ReturnType>System.Func&lt;T&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Provides a mechanism to create an instance of the class or struct when deserializing, using a parameterless constructor.</summary>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonObjectInfoValues`1.xml:100]:
```suggestion
        <summary>Gets or sets a mechanism to create an instance of the class or struct using a parameterless constructor during deserialization.</summary>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonObjectInfoValues`1.xml [hunk 7]
```diff
@@ -118,7 +118,7 @@
         <ReturnType>System.Func&lt;System.Object[],T&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Provides a mechanism to create an instance of the class or struct when deserializing, using a parameterized constructor.</summary>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonObjectInfoValues`1.xml:121]:
```suggestion
        <summary>Gets or sets a mechanism to create an instance of the class or struct using a parameterized constructor during deserialization.</summary>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonObjectInfoValues`1.xml [hunk 8]
```diff
@@ -139,7 +139,7 @@
         <ReturnType>System.Func&lt;System.Text.Json.Serialization.JsonSerializerContext,System.Text.Json.Serialization.Metadata.JsonPropertyInfo[]&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Provides a mechanism to initialize metadata for properties and fields of the class or struct.</summary>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonObjectInfoValues`1.xml:142]:
```suggestion
        <summary>Gets or sets a mechanism to initialize metadata for properties and fields of the class or struct.</summary>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonObjectInfoValues`1.xml [hunk 9]
```diff
@@ -160,7 +160,7 @@
         <ReturnType>System.Action&lt;System.Text.Json.Utf8JsonWriter,T&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Provides a serialization implementation for instances of the class or struct which assumes options specified by <see cref="T:System.Text.Json.Serialization.JsonSourceGenerationOptionsAttribute" />.</summary>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonObjectInfoValues`1.xml:163]:
```suggestion
        <summary>Gets or sets a serialization implementation for instances of the class or struct that assumes options specified by <see cref="T:System.Text.Json.Serialization.JsonSourceGenerationOptionsAttribute" />.</summary>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonParameterInfoValues.xml [hunk 10]
```diff
@@ -51,7 +51,7 @@
         <ReturnType>System.Object</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The default value of the parameter.</summary>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonParameterInfoValues.xml:54]:
```suggestion
        <summary>Gets or sets the default value of the parameter.</summary>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonParameterInfoValues.xml [hunk 11]
```diff
@@ -72,7 +72,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Whether a default value was specified for the parameter.</summary>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonParameterInfoValues.xml:75]:
```suggestion
        <summary>Gets or sets a value that specifies whether a default value was specified for the parameter.</summary>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonParameterInfoValues.xml [hunk 12]
```diff
@@ -93,7 +93,7 @@
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The name of the parameter.</summary>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonParameterInfoValues.xml:96]:
```suggestion
        <summary>Gets or sets the name of the parameter.</summary>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonParameterInfoValues.xml [hunk 13]
```diff
@@ -114,7 +114,7 @@
         <ReturnType>System.Type</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The type of the parameter.</summary>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonParameterInfoValues.xml:117]:
```suggestion
        <summary>Gets or sets the type of the parameter.</summary>
```

File: xml/System.Text.Json.Serialization.Metadata/JsonParameterInfoValues.xml [hunk 14]
```diff
@@ -135,7 +135,7 @@
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The zero-based position of the parameter in the formal parameter list.</summary>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization.Metadata/JsonParameterInfoValues.xml:138]:
```suggestion
        <summary>Gets or sets the zero-based position of the parameter in the formal parameter list.</summary>
```

File: xml/System.Text.Json.Serialization/JsonConverter`1.xml [hunk 15]
```diff
@@ -184,12 +184,20 @@ For more information, see [How to write custom converters for JSON serialization
...
       <Docs>
-        <param name="reader">To be added.</param>
-        <param name="typeToConvert">To be added.</param>
-        <param name="options">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="reader">The <see cref="T:System.Text.Json.Utf8JsonReader" /> to read from.</param>
+        <param name="typeToConvert">The <see cref="T:System.Type" /> being converted.</param>
+        <param name="options">The <see cref="T:System.Text.Json.JsonSerializerOptions" /> being used.</param>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization/JsonConverter`1.xml:189]:
```suggestion
        <param name="options">The options to use when reading the value.</param>
```

File: xml/System.Text.Json.Serialization/JsonConverter`1.xml [hunk 16]
```diff
@@ -184,12 +184,20 @@ For more information, see [How to write custom converters for JSON serialization
...
+        <summary>Reads a dictionary key from a JSON property name.</summary>
+        <returns>The value that was converted.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Method should be overridden in custom converters of types used in deserialized dictionary keys.
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization/JsonConverter`1.xml:197]:
```suggestion
This method should be overridden in custom converters of types used in deserialized dictionary keys.
```

File: xml/System.Text.Json.Serialization/JsonConverter`1.xml [hunk 17]
```diff
@@ -254,11 +262,19 @@ For more information, see [How to write custom converters for JSON serialization
         <Parameter Name="options" Type="System.Text.Json.JsonSerializerOptions" Index="2" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="writer">To be added.</param>
-        <param name="value">To be added.</param>
-        <param name="options">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="writer">The <see cref="T:System.Text.Json.Utf8JsonWriter" /> to write to.</param>
+        <param name="value">The value to convert. Note that the value of <see cref="P:System.Text.Json.Serialization.JsonConverter`1.HandleNull" /> determines if the converter handles <see langword="null" /> values.</param>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization/JsonConverter`1.xml:266]:
```suggestion
        <param name="value">The value to convert. The value of <see cref="P:System.Text.Json.Serialization.JsonConverter`1.HandleNull" /> determines if the converter handles <see langword="null" /> values.</param>
```

File: xml/System.Text.Json.Serialization/JsonConverter`1.xml [hunk 18]
```diff
@@ -254,11 +262,19 @@ For more information, see [How to write custom converters for JSON serialization
...
+        <param name="options">The <see cref="T:System.Text.Json.JsonSerializerOptions" /> being used.</param>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization/JsonConverter`1.xml:267]:
```suggestion
        <param name="options">The options to use when writing the value.</param>
```

File: xml/System.Text.Json.Serialization/JsonConverter`1.xml [hunk 19]
```diff
@@ -184,12 +184,20 @@ For more information, see [How to write custom converters for JSON serialization
...
       </Parameters>
       <Docs>
-        <param name="reader">To be added.</param>
-        <param name="typeToConvert">To be added.</param>
-        <param name="options">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="reader">The <see cref="T:System.Text.Json.Utf8JsonReader" /> to read from.</param>
+        <param name="typeToConvert">The <see cref="T:System.Type" /> being converted.</param>
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization/JsonConverter`1.xml:188]:
```suggestion
        <param name="typeToConvert">The type to convert.</param>
```

File: xml/System.Text.Json.Serialization/JsonConverter`1.xml [hunk 20]
```diff
@@ -254,11 +262,19 @@ For more information, see [How to write custom converters for JSON serialization
...
+        <param name="writer">The <see cref="T:System.Text.Json.Utf8JsonWriter" /> to write to.</param>
+        <param name="value">The value to convert. Note that the value of <see cref="P:System.Text.Json.Serialization.JsonConverter`1.HandleNull" /> determines if the converter handles <see langword="null" /> values.</param>
+        <param name="options">The <see cref="T:System.Text.Json.JsonSerializerOptions" /> being used.</param>
+        <summary>Writes a dictionary key as a JSON property name.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Method should be overridden in custom converters of types used in serialized dictionary keys.
```
Comment (gewarren) [inline xml/System.Text.Json.Serialization/JsonConverter`1.xml:274]:
```suggestion
This method should be overridden in custom converters of types used in serialized dictionary keys.
```

**Review by gewarren — APPROVED** [review]

**Comment by eiriktsarpalis** [comment]
@carlossanlop shouldn't this also be ported to `main`?

**Comment by carlossanlop** [comment]
@eiriktsarpalis no, `main` is not yet targeting RC1. The latest APIs only exist in the `net6-rc1` branch, which is the one we are currently using for the last 6.0 docs, and we will eventually merge that branch into `main`.

**Pull request merged**